resources/lang/id/artist.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'page_description' => 'Artis unggulan di osu!',
+    'title' => 'Artis Unggulan',
+
+    'admin' => [
+        'hidden' => 'ARTIS SAAT INI TERSEMBUNYI',
+    ],
+
+    'beatmaps' => [
+        '_' => 'Beatmaps',
+        'download' => 'Unduh Template Beatmap',
+        'download-na' => 'Template Beatmap belum tersedia',
+    ],
+
+    'index' => [
+        'description' => 'Artis unggulan adalah para artis yang berkolaborasi dengan kami dalam rangka membawa musik yang baru dan original ke osu!. Artis-artis ini dan beberapa lagu mereka yang telah dipilih sendiri oleh tim osu! dengan karyanya yang menakjubkan dan cocok untuk dipakai mapping. Beberapa artis unggulan ini juga telah membuat lagu baru eksklusif untuk digunakan di osu! <br><br>Semua lagu di bagian ini disediakan dalam format .osz yang telah diatur dan telah secara resmi dilisensikan untuk digunakan dalam osu! dan konten yang terkait dengan osu!',
```

Comment:
Uhh, try this for better wording perhaps : 

"Artis Unggulan (Featured Artist) merupakan para artis yang berkolaborasi dengan kami dalam rangka membawa musik yang baru dan original ke osu!. Artis-artis berikut beserta lagu-lagu di bawah ini telah secara khusus diseleksi oleh tim osu! agar sekiranya cocok untuk dapat di-map dengan kesan yang mewah. Beberapa dari mereka bahkan telah membuat lagu-lagu baru yang secara eksklusif dibuat untuk osu!.<br><br>Semua lagu yang disediakan di bawah ini tersedia untuk diunduh dalam format .osz dengan timing yang sudah disediakan sebelumnya dan telah secara resmi dilisensikan untuk penggunaan di dalam osu! dan konten-konten lainnya yang terkait dengan osu!"